include/librealsense2/h/rs_device.h

```diff
@@ -146,6 +146,53 @@ void rs2_connect_tm2_controller(const rs2_device* device, const unsigned char* m
 */
 void rs2_disconnect_tm2_controller(const rs2_device* device, int id, rs2_error** error);
 
+/**
+* Set tm2 camera intrinsics
+* \param[in] device       tm2 device
+* \param[in] sensor_id    specify fisheye lens
+* \param[in] intr         intrinsics value to be written to the device
+* \param[out] error       If non-null, receives any error that occurs during this call, otherwise, errors are ignored
+* \return 0 if the operation is completed successfully.
+*/
+void rs2_set_tm2_intrinsics(const rs2_device* device, int sensor_id, const rs2_intrinsics* intr, rs2_error** error);
+
+/**
+   * Set tm2 camera extrinsics
```

Comment:
All instances of  "tm2" to be omitted  int the public API, see above

Reply:
All gone now